index.html

```diff
@@ -4,10 +4,96 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
     <link rel='stylesheet' href='./style.css' />
     <title>Component Library</title>
 </head>
 <body>
+
+    <h1>Badges</h1>
+    <h3>notification badges</h3>
+    <div class='badge-size-sm badge-green border-radius-circle'></div>
+    <div class='badge-size-sm badge-red border-radius-circle'></div>
+    <div class='badge-size-sm badge-yellow border-radius-circle'></div>
+    <div class='badge-size-sm badge-blue border-radius-circle'></div>
+    <br/>
+    <div class='badge-size-md badge-green border-radius-circle'></div>
+    <div class='badge-size-md badge-red border-radius-circle'></div>
+    <div class='badge-size-md badge-yellow border-radius-circle'></div>
+    <div class='badge-size-md badge-blue border-radius-circle'></div>
+    <br/>
     
+    <h3>text only badges</h3>
+    <div class='badge-size-sm badge-text-sm flex-center padding-sm border-radius-circle badge-blue'>
+        4
+    </div><br/>
+
+    <h3>badges on icon</h3>
+    <div class='inline-block pos-relative'>
+        <span class='material-icons'>account_balance</span>
+        <div class='badge-size-sm badge-text-sm flex-center border-radius-circle badge-blue pos-absolute top-left-60'>1</div>
```

Comment:
Personal Opinion : The color contrast makes it hard to read the text on the badge. It would be nice if this could be altered say white text on blue badge.

Reply:
Yes, nice suggestion. i'll make the changes. thanks.